### ISSUE TITLE
Missing requirements.txt

### ISSUE THREAD

**Issue by MaggiePas** [pyaf/DenseNet-MURA-PyTorch#12]
Install would be much easier with a requirements.txt file

This can be easily done with:
`pip freeze > requirements.txt`

There are required packages missing from the listed dependencies is the README.md

Thanks!

**Comment by pyaf** [comment]
Yup! missed that one, I do remember it was PyTorch 0.3, not sure of others. If you are able to run this project, do me a favor open a PR adding a `requirements.txt` file :)
